### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jeswan**
This PR combines the old `jiant` README with the new `jiant-dev` README and adds additional documentation to introduce new users to `jiant`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +54,24 @@ If you use `jiant` in academic work, please cite it directly:
...
+- [What do you learn from context? Probing for sentence structure in contextualized word representations](https://openreview.net/forum?id=SJzSgnRcKX) ("edge probing")
+- [BERT Rediscovers the Classical NLP Pipeline](https://arxiv.org/abs/1905.05950) ("BERT layer paper")
+- [Probing What Different NLP Tasks Teach Machines about Function Word Comprehension](https://arxiv.org/abs/1904.11544) ("function word probing")
+- [Investigating BERT’s Knowledge of Language: Five Analysis Methods with NPIs](https://arxiv.org/abs/1909.02597) ("BERT NPI paper")
+
+To exactly reproduce experiments from [the ELMo's Friends paper](https://arxiv.org/abs/1812.10860) use the [`jsalt-experiments`](https://github.com/jsalt18-sentence-repl/jiant/tree/jsalt-experiments) branch. That will contain a snapshot of the code as of early August, potentially with updated documentation.
+
+For the [edge probing paper](https://openreview.net/forum?id=SJzSgnRcKX) and the [BERT layer paper](https://arxiv.org/abs/1905.05950), see the [probing/](probing/) directory.
```
Comment (jeswan) [inline README.md:70]: Note to include branch here

File: README.md [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
 
-`jiant` is a software toolkit for natural language processing research, designed to facilitate work on multitask learning and transfer learning for sentence understanding tasks. 
+</div>
+
+**Why should I use `jiant`?**
+- `jiant` supports [multitask learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_Multi_Task_Example.ipynb)
+- `jiant` supports [transfer learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_STILTs_Example.ipynb)
+- `jiant` supports these natural language understanding [tasks](./md_docs/tasks.md)
+- `jiant` supports these benchmarks:
+    - [GLUE](https://gluebenchmark.com) (TODO)
```
Comment (sleepinyourhat) [inline README.md:19]: If these TODOs will be there for the release, explain that it's only the tutorial, not the implementation, that's pending.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,48 @@
...
+- `jiant` is configuration file driven
+- `jiant` is built with [PyTorch](https://pytorch.org)
+- `jiant` integrates with [datasets](https://github.com/huggingface/datasets) to manage task data and uses HuggingFace Transformers [implementations](https://github.com/huggingface/transformers) to manage models and tokenizers.
+
 
 ### Installation
 ```
-git clone https://github.com/jiant-dev/jiant.git
-pip install -r jiant/requirements-dev.txt
+pip install jiant
```
Comment (sleepinyourhat) [inline README.md:32]: Are we encouraging people to do this even if they plan to contribute to Git?

File: README.md [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
-### Issues
-Templates for bug reports and feature requests are provided [here](https://github.com/jiant-dev/jiant/issues/new/choose).
-
 ### Contributing
 The `jiant` project's contributing guidelines can be found [here](CONTRIBUTING.md).
 
-### License
-`jiant` is released under the [MIT License](https://github.com/jiant-dev/jiant/blob/master/LICENSE).
+### Looking for a previous version of `jiant`?
+The previous version of `jiant` has been moved to [jiant-v1-legacy](https://github.com/nyu-mll/jiant-v1-legacy) although history is available in this repository. The current version of `jiant` is more modular, scalable, and designed to reflect the needs of the current NLP research community.
```
Comment (sleepinyourhat) [inline README.md:43]: Name 1.3.x?

File: README.md (hunk 2 above)
Comment (jeswan) [reply README.md:19]: These TODOs will be removed before opening a PR and updated to reflect the benchmark documentation location

File: README.md [hunk 5]
```diff
@@ -1,30 +1,48 @@
...
+[![Generic badge](https://img.shields.io/github/v/release/nyu-mll/jiant)](https://shields.io/)
+[![codecov](https://codecov.io/gh/nyu-mll/jiant/branch/master/graph/badge.svg)](https://codecov.io/gh/nyu-mll/jiant)
+[![CircleCI](https://circleci.com/gh/nyu-mll/jiant/tree/master.svg?style=shield)](https://circleci.com/gh/nyu-mll/jiant/tree/master)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-`jiant` is a software toolkit for natural language processing research, designed to facilitate work on multitask learning and transfer learning for sentence understanding tasks. 
+</div>
+
+**Why should I use `jiant`?**
```
Comment (jeswan) [inline README.md:14]: @zphang I'm looking for a concise statement about why people should use `jiant` over `transformers` for training/evaluation. Phil mentioned that you found the "seams" in `transformers` that caused problems with research. Do you think you can capture this in a bullet point?

File: README.md [hunk 6]
```diff
@@ -1,30 +1,48 @@
...
+- `jiant` supports [multitask learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_Multi_Task_Example.ipynb)
+- `jiant` supports [transfer learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_STILTs_Example.ipynb)
+- `jiant` supports these natural language understanding [tasks](./md_docs/tasks.md)
+- `jiant` supports these benchmarks:
+    - [GLUE](https://gluebenchmark.com) (TODO)
+    - [SuperGLUE](https://super.gluebenchmark.com/) (TODO)
+    - [XTREME](https://sites.research.google/xtreme/) by following [these steps](./jiant/scripts/postproc/xtreme/README.md)
```
Comment (jeswan) [inline README.md:21]: Add note about models

File: README.md [hunk 7]
```diff
@@ -1,30 +1,48 @@
...
+- `jiant` is a research library and users are encouraged to extend, change, and contribute to match their needs!
+
+**A few additional things you might want to know about `jiant`:**
+- `jiant` is configuration file driven
+- `jiant` is built with [PyTorch](https://pytorch.org)
+- `jiant` integrates with [datasets](https://github.com/huggingface/datasets) to manage task data and uses HuggingFace Transformers [implementations](https://github.com/huggingface/transformers) to manage models and tokenizers.
```
Comment (jeswan) [inline README.md:27]: combine

File: README.md [hunk 8]
```diff
@@ -1,30 +1,48 @@
...
 ```
-git clone https://github.com/jiant-dev/jiant.git
-pip install -r jiant/requirements-dev.txt
+pip install jiant
 ```
-You can  confirm that `jiant` is installed by running a basic experiment using the instructions [here](./examples/guides/quick_start_cli.md).
+You can confirm that `jiant` is installed by running a basic [experiment](./examples/simple_api_fine_tuning.ipynb).
 
 ### Getting started
 Examples to help you get started can be found here [here](./examples/README.md).
```
Comment (jeswan) [inline README.md:37]: Include simple example here

File: README.md [hunk 9]
```diff
@@ -1,30 +1,48 @@
...
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-`jiant` is a software toolkit for natural language processing research, designed to facilitate work on multitask learning and transfer learning for sentence understanding tasks. 
+</div>
+
+**Why should I use `jiant`?**
+- `jiant` supports [multitask learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_Multi_Task_Example.ipynb)
+- `jiant` supports [transfer learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_STILTs_Example.ipynb)
+- `jiant` supports these natural language understanding [tasks](./md_docs/tasks.md)
```
Comment (jeswan) [inline README.md:17]: Highlight esoteric tasks such as edge probing?

**Pull request merged**